### DISCUSSION TITLE
Map number shortcuts to both number bar and number pad

### DISCUSSION THREAD

**Discussion by idahotallpaul** [rxhanson/Rectangle#1029] [Ideas]
Hi. I have an Apple 128 key keyboard I use at my desk and I have the Rectangle shortcuts mapped to the number pad at the right. This works super well because the number pad is easier to navigate by feel than the number bar. When I'm working on just my laptop though, I lose access to all Rectangle window management. I've been dealing with this by firing up divvy when I'm in laptop mode, but this is less than ideal.

I'm wondering if shortcuts containing numerals could be mapped by default to to both the number pad and the number bar? When you look at your preferences, it makes no distinction between a `^⇧1` triggered by a number bar and one triggered by a number pad. There is no distinction in the output as you type either, so intuitively, I'd expect key combinations with either to function the same. 

If we can't enable that by default, could we get a preference added to enable that mapping?

Thanks.

**Comment by rxhanson** [comment]
Thanks for the feedback. I don't have the bandwidth to add this in, but would merge a good pull request with this as a preference (terminal command enabled, unless there's large enough demand for something in the UI).
